### ISSUE TITLE
ORM events ? 

### ISSUE THREAD

**Issue by jaumard** [trailsjs/trailpack-realtime#4]
It will be nice to create a room for each models and have a created/updated/deleted event from the ORM.

In order to do this each trailpack-orm should fire an event `orm-model-created` `orm-model-updated` and `orm-model-deleted` on his lifecycle event and this trailpack have to listen for those events and send them to sockets rooms.

@konstantinzolotarev @tjwebb @wbprice @weyj4 any feedback on this ?

**Comment by konstantinzolotarev** [comment]
@jaumard I agree with this approach but think that it should be optional. There should be option to disable it.

**Comment by jaumard** [comment]
Of course :) we can add a boolean in config to plug `trailpack-realtime` to ORM events or not

**Comment by scott-wyatt** [comment]
+1 for this functionality.
